packages/mgt/src/graph/graph.people.ts

```diff
@@ -160,6 +160,49 @@ export async function findPeople(
   return graphResult ? graphResult.value : null;
 }
 
+/**
+ * async promise, returns all matching Graph users who are member of the specified group
+ *
+ * @param {string} query
+ * @param {string} groupId - the group to query
+ * @param {number} [top=10] - number of people to return
+ * @param {PersonType} [personType=PersonType.person] - the type of person to search for
+ * @returns {(Promise<Person[]>)}
+ */
+export async function findPeopleFromGroup(
+  graph: IGraph,
+  query: string,
+  groupId: string,
+  top: number = 10,
+  personType: PersonType = PersonType.person
+): Promise<Person[]> {
```

Comment:
The members api returns Users (instead of People) I believe, so the return type here should be `User` instead of `Person`. This function should then be renamed to `findUsersFromGroup` and moved to `graph.user.ts`. Likewise, the users cache should be used.

Reply:
Fully agree. Will make the changes!